src/Raven.Server/RavenServer.cs

```diff
@@ -313,9 +319,7 @@ public void RefreshClusterCertificate(object state)
                 using (context.OpenReadTransaction())
                 {
                     var certUpdate = ServerStore.Cluster.GetItem(context, "server/cert");
-                    if (certUpdate != null
-                        && certUpdate.TryGet("Thumbprint", out string thumbprint)
-                        && thumbprint != currentCertificate.Certificate.Thumbprint)
+                    if (certUpdate != null)
```

Comment:
comment explicitly explaining the logic?

Reply:
@ayende actually when looking at this again, the logic is wrong... we will always return here starting after the first replacement because we don't delete "server/cert". Should we always send the recheck command?

Reply:
Yes, I know.
That is why I said we will have it once an hour.
Once the cert is replaced, that is fine to do.

However, need to check that the `server/cert` is the more up to date version ?